bin/css-validator

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+`echo eval exec node $(printf %q\  "$0" "$@")`
+const fs = require('fs')
+const validateCss = require('css-validator')
+
+validateCss({text: fs.readFileSync(process.argv[2])}, (err, data) => {
```

Comment:
We aren't guaranteed that `process.argv[2]` will exist on execution. Can we use something like `commander` (personal preference) or `yargs` (might be necessary for repetition) so it presents a `--help` dialog on a missing filename?

For v1, we are probably fine with:

```
css-validator <filename> <filename...> (unsure about syntax)
```

For later revisions, we'll probably want directory support (e.g. `bin/`) which autorecurses